src/Maestro/Maestro.Common/AppCredentialResolver.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Azure.Core;
+
+namespace Microsoft.DotNet.Maestro.Common;
+
+public static class AppCredentialResolver
+{
+
+    public static TokenCredential CreateTokenCredential(
```

Comment:
```suggestion
    public static TokenCredential CreateCredential(
```

or possibly `CreateAppCredential`

Reply:
Wanted it to be closer to the returned type, but `CreateCredential` sounds better. Thanks, will fix